Three/include/CGAL/Three/Scene_interface.h

```diff
@@ -45,13 +45,13 @@ class Scene_group_item;
  */
 enum RenderingMode { Points = 0,
                      PointsPlusNormals,
+                     ShadedPoints,
                      Splatting,
                      Wireframe, 
                      Flat,
                      FlatPlusEdges,
                      Gouraud,
-                     LastRenderingMode = Gouraud,
-                     NumberOfRenderingMode = LastRenderingMode+1 };
+                     NumberOfRenderingMode};
```

Comment:
NumberOfRenderingMode is used by Scene_item, doesn't it mean that every item will have to be rebuilt anyway ?